From 9d36d571ee7b94a62231aadac4b08eb266612596 Mon Sep 17 00:00:00 2001
From: Stefan Haller <stefan@haller-berlin.de>
Date: Sat, 29 Mar 2025 17:58:00 +0100
Subject: [PATCH 03/21] gitk: add catch around the code that brings gitk to the
 front

We do this for git-gui too, so there must be some reason.
---
 gitk-git/gitk | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/gitk-git/gitk b/gitk-git/gitk
index 53162c139f..5747b56ed4 100755
--- a/gitk-git/gitk
+++ b/gitk-git/gitk
@@ -12453,11 +12453,13 @@ namespace import ::msgcat::mc
 
 # on OSX bring the current Wish process window to front
 if {[tk windowingsystem] eq "aqua"} {
-    safe_exec [list osascript -e [format {
-        tell application "System Events"
-            set frontmost of processes whose unix id is %d to true
-        end tell
-    } [pid] ]]
+    catch {
+        safe_exec [list osascript -e [format {
+            tell application "System Events"
+                set frontmost of processes whose unix id is %d to true
+            end tell
+        } [pid] ]]
+    }
 }
 
 # Unset GIT_TRACE var if set
-- 
2.51.0.188.g7229e14d4b.dirty

